easybuild/easyconfigs/m/M4/M4-1.4.17.eb

```diff
@@ -4,19 +4,25 @@ name = 'M4'
 version = '1.4.17'
 
 homepage = 'http://www.gnu.org/software/m4/m4.html'
-description = """GNU M4 is an implementation of the traditional Unix macro processor.
- It is mostly SVR4 compatible although it has some extensions 
+
+description = """
+ GNU M4 is an implementation of the traditional Unix macro processor.
+ It is mostly SVR4 compatible although it has some extensions
  (for example, handling more than 9 positional parameters to macros).
- GNU M4 also has built-in functions for including files, running shell commands, doing arithmetic, etc.
+ GNU M4 also has built-in functions for including files, running shell
+ commands, doing arithmetic, etc.
 """
 
 toolchain = {'name': 'dummy', 'version': 'dummy'}
 
 sources = [SOURCELOWER_TAR_GZ]
 source_urls = [GNU_SOURCE]
 
-# '-fgnu89-inline' is required to avoid linking errors with older glibc's,
-# see https://github.com/hpcugent/easybuild-easyconfigs/issues/529
+osdependencies = [
+    ('gcc'),
+    ('glibc-devel'),
```

Comment:
@JackPerdue hmm, wouldn't it be better to add a check for these in the EasyBuild framework itself, since this is basically the assumption whenever the `dummy` toolchain is used?